iConverter/Pages/Main/MainViewController.swift

```diff
@@ -6,66 +6,89 @@
 //
 
 import UIKit
-import SwinjectAutoregistration
 import RxSwift
+import RxOptional
 
 class MainViewController: BaseViewController {
-    
-    // MARK: - Outlets
-    @IBOutlet weak var currentBalanceTitle: UILabel!
-    @IBOutlet weak var currentBalance: UILabel!
-    @IBOutlet weak var balancesContainer: BalancesContainer!
-    @IBOutlet weak var historyContainer: UIView!
-    @IBOutlet weak var historyTableVeiw: UITableView!
-    
+
+    // MARK: - view
+    var mainView: MainView! {
```

Comment:
Please make this private and I think we don't need the `didSet`, just call `configureTableView` in `viewDidLoad` since `loadView` is called before that